x/staking/keeper/determinstic_test.go

```diff
@@ -0,0 +1,945 @@
+package keeper_test
```

Comment:
Can we move `tests/integration/bank/deterministic_test.go` under `x/bank/keeper` if that's where the deterministic tests go? Otherwise, let's move this one under the integration tests.

Reply:
Actually, up on this one: let's move it to `test/integration`.

@atheeshp you can put automerge on after this small change 👍

Reply:
ah, missed it.